issuance/message/example/credential-request-message.json

```
@@ -0,0 +1,7 @@
+{
+    "@context": "http://example.org/context",
+    "type": "CredentialRequestMessage",
+    "format": "ldp_vc",
+    "types": ["VerifiableCredential", "UniversityDegreeCredential"]
```

```suggestion
    "type": ["VerifiableCredential", "UniversityDegreeCredential"]
```
please check across schema, shape, sample

---

done